css/templates/e-commerce/page.css

```diff
@@ -125,6 +125,62 @@ dd:after {
   display: block;
 }
 
+
+/* CUSTOM LOADER */
+.commerce-loader,
+.commerce-loader::before,
+.commerce-loader::after {
+  border-radius: 50%;
+  width: var(--space-1);
+  height: var(--space-1);
+  animation-fill-mode: both;
+  animation: commerce-loader 1.44s infinite ease-in-out;
+  will-change: contents;
+}
+
+.commerce-loader {
+  color: var(--color-secondary);
+  margin: 24% auto;
+  animation-delay: .16s;
+}
+
+.commerce-loader::before,
+.commerce-loader::after {
+  content: '';
+  position: absolute;
+  top: 0;
+}
+
+.commerce-loader::before {
+  left: -1rem;
```

Comment:
Using -1rem explicitly here and in keyframes instead of calc(-1 * var(--space-2)) to optimize performance/prevent the browser from calculating.